src/index.js

```diff
@@ -1,13 +1,44 @@
-export default function BaseballGame() {
-  this.play = function (computerInputNumbers, userInputNumbers) {
-    return "결과 값 String";
-  };
-}
-
-// export default class BaseballGame {
-//   play(computerInputNumbers, userInputNumbers) {
-//     return "결과 값 String";
-//   }
-// }
-
-new BaseballGame();
+import createComputerNum from './createComputerNum.js';
+import { showSuggestion, hideSuggestion, printResult } from './handleElement.js';
+import { getResult, getUserInput, checkGameWin, formatResult } from './utils.js';
+
+// 게임을 진행할 변수 데이터
+const numArray = [null, null];
```

Comment:
변수데이터를 단순히 Array에 담아서 관리하게 되면 코드를 볼 때 0번째, 1번째 인덱스에 어떤 값이 들어있고, 어떻게 사용하는지 알기 어려운데요.

차라리 아래와 같이 객체를 사용해보는건 어떨까요? (예시이니 참고만 해주세요~)

```suggestion
// 게임을 진행할 변수 데이터
const gameInfo = {
   computerNum: null,
   userNum: null
}
```